### PR TITLE
Doc: Updating Database Connector Docs for SSL

### PR DESCRIPTION
**Pull request by Prajwal214**
### Describe your changes:

Fixes <issue-number>

Updating Database Connector Docs for SSL

#
### Type of change:

- [ ] Bug fix
- [ ] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-docs/content/v1.5.x-SNAPSHOT/connectors/database/doris/index.md [hunk 1]
```diff
@@ -65,6 +66,16 @@ serviceConnectionPath: "/images/v1.5/connectors/doris/service-connection.png",
 
 {% /stepsContainer %}
 
+## Securing Doris Connection with SSL in OpenMetadata
+
+To establish secure connections between OpenMetadata and Doris, navigate to the `Advanced Config` section. Here, you can provide the CA certificate used for SSL validation by specifying the `caCertificate`. Alternatively, if both client and server require mutual authentication, you'll need to use all three parameters: `ssl_key`, `ssl_cert`, and `ssl_ca`. In this case, `ssl_cert` is used for the client’s SSL certificate, `ssl_key` for the private key associated with the SSL certificate, and `ssl_ca` for the CA certificate to validate the server’s certificate.
+
+{% image
+  src="/images/v1.4/connectors/ssl_connection.png"
```
Comment (SumanMaharana) [inline openmetadata-docs/content/v1.5.x-SNAPSHOT/connectors/database/doris/index.md:74]: should we use the images under `v1.5/connectors` here as we have the same images under `v1.5` as well ?

File: openmetadata-docs/content/v1.5.x-SNAPSHOT/connectors/database/greenplum/index.md [hunk 2]
```diff
@@ -137,6 +138,24 @@ In order to integrate SSL in the Metadata Ingestion Config, the user will have t
...
+Under `Advanced Config`, specify the SSL mode appropriate for your connection, such as `prefer`, `verify-ca`, `allow`, and others. After selecting the SSL mode, provide the CA certificate used for SSL validation (`caCertificate`). Note that Greenplum requires only the CA certificate for SSL validation.
+
+{% note %}
+
+For IAM authentication, it is recommended to choose the `allow` mode or another SSL mode that fits your specific requirements.
+
+{% /note %}
+
+{% image
+  src="/images/v1.4/connectors/ssl_connection.png"
```
Comment (SumanMaharana) [inline openmetadata-docs/content/v1.5.x-SNAPSHOT/connectors/database/greenplum/index.md:154]: here and other places as well  for `v1.5`.

Comment (Prajwal214) [reply openmetadata-docs/content/v1.5.x-SNAPSHOT/connectors/database/greenplum/index.md]: @SumanMaharana Update it to v.1.5.

File: openmetadata-docs/content/v1.5.x-SNAPSHOT/connectors/database/doris/index.md (hunk 1 above)
Comment (Prajwal214) [reply openmetadata-docs/content/v1.5.x-SNAPSHOT/connectors/database/doris/index.md]: Updated the path to v1.5

**Review by SumanMaharana — APPROVED** [review]

**Review by ayush-shah — APPROVED** [review]

**Pull request merged**